internal/app/feed/export_feeds.go

```diff
@@ -9,6 +9,33 @@ import (
 	"github.com/spf13/viper"
 )
 
+func CreateSchemas(exporter Exporter) error {
+	logger := util.GetLogger()
+	logger.Info("Creating schemas started")
+
+	feeds := []Feed{
```

Comment:
Nice! We should be able to apply this to Export feeds as well. Would be a lot nicer...

Reply:
I think we can create something like this and iterate over it: 
```
feeds := map[string]Feed {
   "inspections": &InspectionFeed{...},
   ...
}
```

Reply:
Perhaps, the `Feed` also has a `Name()` method which you can use to compare agains the enabled map.

Reply:
You're right, we can use the `Name()` instead. Do you want me to make the change or should we do it in a separate PR?

Reply:
Let's do it separately. No need to bring it into this PR.